### DISCUSSION TITLE
Lexical Error。

### DISCUSSION THREAD

**Discussion by JosephC666** [JSQLParser/JSqlParser#1772] [Q&A] [answered]
Hello~
When there is Unicode in my query, an error “Lexical error at line 1…” will occur. Can you push a new version that supports Unicode?
Query:
Select 玻璃=1500;

use version: 4.6

Thanks!

**Comment by manticore-projects** [comment, marked as answer]
Greetings,

please use latest SNAPSHOT from GIT master, which includes PR #1740. It has been fixed already.
